src/pynwb/ndx_sound/widgets.py

```diff
@@ -263,4 +263,4 @@ def acoustic_waveform_widget(time_series: TimeSeries, time_window=None, **kwargs
     )
 
 
-default_neurodata_vis_spec.update({AcousticWaveformSeries: acoustic_waveform_widget})
+default_neurodata_vis_spec.update({AcousticWaveformSeries: AcousticWaveformWidget})
```

Comment:
Tried to change the default visualization to the new widget, it doesn't seem to have effect.

```
stimulus: {
    AcousticWaveformSeries <class 'abc.AcousticWaveformSeries'>
  }
```

```python
from ndx_sound import widgets
from nwbwidgets import nwb2widget

nwb2widget(nwbfile)
```

<img width="992" alt="Screenshot 2022-08-26 at 13 54 32" src="https://user-images.githubusercontent.com/24475788/186899171-b7838667-7004-4452-a41f-578ca11ec2ec.png">

@bendichter do you have any idea why?

Reply:
Still no success in overriding the default `default_neurodata_vis_spec`:

```python
from nwbwidgets import nwb2widget
from ndx_sound.widgets import load_widgets

load_widgets()
nwb2widget(nwbfile)
```

Hoewever in `default_neurodata_vis_spec` it seems to be okay:
```
{...,
abc.AcousticWaveformSeries: ndx_sound.widgets.AcousticWaveformWidget}
```

Reply:
@CodyCBakerPhD Thank you for helping figuring out the final solution!
The solution was to load the nwb file within the same cell in a jupyter notebook, just like this:

```python
from pynwb import NWBHDF5IO
from ndx_sound.widgets import load_widgets
from nwbwidgets import nwb2widget

# load the audio widget into nwbwidgets
load_widgets()

io = NWBHDF5IO("test_with_only_audio.nwb", mode="r", load_namespaces=True)
nwbfile = io.read()
nwb2widget(nwbfile)

```
@CodyCBakerPhD this also works when `load_widgets` have a local import of `default_neurodata_vis_spec`, I just tried out!

<img width="1145" alt="Screenshot 2022-09-06 at 18 08 56" src="https://user-images.githubusercontent.com/24475788/188684130-b09353c9-dfdc-464d-91a8-6a8ef38bb041.png">